Modules/_io/textio.c

```diff
@@ -309,8 +311,9 @@ _PyIncrementalNewlineDecoder_decode(PyObject *myself,
     nldecoder_object *self = (nldecoder_object *) myself;
 
     if (self->decoder == NULL) {
-        PyErr_SetString(PyExc_ValueError,
-                        "IncrementalNewlineDecoder.__init__ not called");
+        PyErr_Format(PyExc_ValueError,
+                     "%.200s.__init__() not called",
```

Comment:
Revert this change. The problem is that `IncrementalNewlineDecoder.__init__` not called. This can happen if subclass' `__init__` doesn't call `super().__init__`. Subclass's `__init__` can be called, but `self->decoder == NULL` if `IncrementalNewlineDecoder.__init__` not called.